test/factories/discovery_host_related.rb

```diff
@@ -1,24 +1,26 @@
-FactoryGirl.define do
-  factory :discovered_host, class: 'Host::Discovered' do
-    sequence(:name) { |n| "host#{n}" }
-    sequence(:ip)   { |n| IPAddr.new(n, Socket::AF_INET).to_s }
-    sequence(:mac)  { |n| "02:23:45:67:" + n.to_s(16).rjust(4, '0').insert(2, ':') }
+unless FactoryGirl.factories.registered?(:discovered_host)
```

Comment:
So when you run tests via `TEST=xyz` it loads files in a different way and factory_girl was giving an error - Factory already registered. I could not find any other way than doing this, I want to be able to run all tests and a particular test.